infra/core/app-configuration/config-data/config-data.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+      "name": "ENVIRONMENT",
+      "value": "#{{ lower(environment) }}",
+      "label": "Platform"
+    },
+    {
+      "name": "ACR_NAME",
+      "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
+      "label": "Platform"
+    },
+    {
+      "name": "NAMESPACE",
+      "value": "flux-config",
+      "label": "Platform"
+    },
+    {
+      "name": "SERVICEBUS_RG",
+      "value": "#{{ servicesResourceGroup }}",
+      "label": "Platform"
+    },
+    {
+      "name": "SERVICEBUS_NS",
+      "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_servicebus }}#{{ nc_instance_regionid }}01",
+      "label": "Platform"
+    },
+    {
+      "name": "POSTGRES_SERVER_RG",
+      "value": "#{{ dbsResourceGroup }}",
+      "label": "Platform"
+    },
+    {
+      "name": "POSTGRES_SERVER",
```

Comment:
Should we have POSTGRES_SERVER_{IDENTIFIER} ?? so if we add many servers etc.. servers for major programme etc?

Reply:
Updated the variable to 1401 where 1401 is the identifier for the current server..

Reply:
Changed to SHARED_POSTGRES_SERVER_01